velox/docs/functions/datetime.rst

```diff
@@ -97,6 +97,10 @@ These functions are supported for TIMESTAMP and DATE values.
 
     Returns the month of the year from ``x``.
 
+.. function:: quarter(x) -> bigint
```

Comment:
Please can you also add an example of date_trunc to quarter (in the description above this).

Reply:
Just added. Thank you for pointing this out!